pages/samvera/developer_resources/collections/hyrax_2.1/nesting-faq.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "FAQ - Collection Nesting"
+keywords: Nesting, Collection, FAQ
+categories: How to Do All the Things
+permalink: collection-nesting-faq.html
+folder: samvera/how-to/collection_extensions/hyrax_2.1/nesting-faq.md
+sidebar: home_sidebar
+tags: [development_resources]
+a-z: ['FAQ - Collection Nesting', 'Collections - Nesting', 'Nesting of Collections']
+version:
+  label: 'targeted for Hyrax v2.1.0'
+  branch:
+    label: 'collections-sprint'
+    link: 'https://github.com/samvera/hyrax/tree/collections-sprint'
+---
+
+### What is collection nesting?
+
+Collections that have collections as members.
+
+### Can a collection have collections and works as members?
+
+Yes.  A collection can have all sub-collections, all works, or a combination of both.
+
+### Is there a limit to how deep the nesting can go?
+
+The depth of nesting is limited. By default the limit is set to 5.  You can modify the limit by...  
+
+<b style='padding-left: 30px; color: purple; font-size: 1.3em;'>TODO: how to change limit?</b>
+
+### Can a collection be its own descendant?
+
+No. An ancestor validation is performed when a collection is added to another collection to prevent circular chains.
+
+### Can collections of different types be nested?
+
+No.  At this time, nesting is limited to collections of the same type.  For example, you can nest a User Collection in a User Collection AND you can nest an Organizational Collection in an Organization Collection, but you can NOT nest a User Collection in an Organizational Collection.
+
+In the future, this could be relaxed to allow a configuration that identifies which collection types can be nested in which, if this is determined to be a priority.
+
+### Where do I set collection nesting?
```

Comment:
This title sounds a bit to me like it could refer to the ability to nest a collection... i.e. the collection type. I would suggest something like "Where do I manage collection nesting relationships?"